### PR TITLE
Various small updates

### PR DESCRIPTION
**Pull request by Lucretiel**
This pull request fixes some of the broken issues we've seen and make other general small updates.

### CODE REVIEW

File: gulpfile.babel.js [hunk 1]
```diff
@@ -288,6 +288,18 @@ export const keycheck = () => (
       })
 );
 
+// eslint-disable-next-line camelcase
+export const bitly_keycheck = () => (
```
Comment (diminishedprime) [inline gulpfile.babel.js]: Doesn't matter, but why is this wrapped in parens? Isn't it just a single expression?

File: gulpfile.babel.js [hunk 2]
```diff
@@ -288,6 +288,18 @@ export const keycheck = () => (
...
+  fs
+      .access('./bitly_api_key.cfg')
+      .catch(err => {
+        throw new Error(
+            err +
+        '\nNeed a bitly key in order to do URL ' +
```
Comment (diminishedprime) [inline gulpfile.babel.js]:
```suggestion
        '\nYou need a bitly key in order to do URL ' +
```

File: gulpfile.babel.js (hunk 1 above)
Comment (Lucretiel) [reply gulpfile.babel.js]: I think this might be an eslint thing? I'll revert and see if it switches back after an eslint pass

Comment (Lucretiel) [reply gulpfile.babel.js]: Doesn't seem like an eslint thing. I think I just did it for consistency with the `{}` arrow functions.

**Pull request merged**